src/services/housing.js

```diff
@@ -55,12 +58,11 @@ import './user';
  * @return {Promise.<Boolean>} True if the user is authorized to view the housing application staff page
  */
 const checkHousingStaff = async () => {
-  return false; //! DEBUG
-  // try {
-  //   return await http.get(`housing/staff`);
-  // } catch {
-  //   return false;
-  // }
+  try {
+    return await http.get(`housing/staff`);
```

Comment:
The staff endpoint has not been made yet, has it?

Reply:
Oh, it's in 440. I will remember this while looking into authorization. I am not sure it fits the way authorization is done on the rest of the site.

Reply:
Yes, this is a feature that is still WIP in the backend, which is why the call is also put into a `try ... catch`, 
(as well as to make sure the page defaults to the student page if something goes wrong under normal operation)

Reply:
More importantly, it is not the main feature of this branch, and it is only implemented as far as it is to allow the existing UI features to continue to work as expected. It will easily be changed in the future when that housing staff authorization aspect is actually ready

Reply:
Great, makes sense